.github/workflows/operate-first-publish.yaml

```diff
@@ -0,0 +1,51 @@
+name: Update Operate First directory
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Update Operate First directory
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: Checkout Operate First repository
+        uses: actions/checkout@v2
+        with:
+          repository: eletonia/apps
+          path: operate-first-repo
```

Comment:
eletonia -> fybrik  This means a fork of apps needs to be created in fybrik?

Reply:
Yes we could change these to the fybrik user
@roee88 What do you think?

Reply:
Sounds good. I forked it to https://github.com/fybrik/apps and made you admin. You may want to rename the fork to something other than `apps` to avoid confusion (I suggest operate-first-apps).